src/css/homepage.css

```diff
@@ -89,3 +140,56 @@
 .homepage-shape-links-container > img {
     width: 10%;
 }
+
+/* Homepage theme park shape CSS & animation */
+
+.homepage-shape-links-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 2em;
+    background-image: linear-gradient(white, white 30%, white 40%, dodgerBlue 100%);
+}
+
+.homepage-shape-links-container > img {
```

Comment:
There's already a reference for this type of CSS above in the file at line `140`, you should apply this padding there too.